### DISCUSSION TITLE
stoicism/william_catton/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#25] [General]
# stoicism/william_catton/

https://shanenull.com/stoicism/william_catton/

**Comment by shane0** [comment]
https://vimeo.com/19357560

**Reply by shane0** [reply]
- what we call production is extraction
- nature produces it
- nature renews it
- to be sustainable means we slow extraction to the rate of natures production
- to be sustainable means we slow pollution to the rate nature can absorb
